src/engine/SCons/Node/FS.py

```diff
@@ -3325,6 +3473,28 @@ def rfile(self):
         self._memo['rfile'] = result
         return result
 
+    def find_repo_file(self):
+        """
+        For this node, find if there exists a corresponding file in one or more repositories
+        :return: list of corresponding files in repositories
+        """
+        retvals = []
+
+        norm_name = _my_normcase(self.name)
+        for repo_dir in self.dir.get_all_rdirs():
+            try:
+                node = repo_dir.entries[norm_name]
+            except KeyError:
+                node = repo_dir.file_on_disk(self.name)
+
+            if node and node.exists() and \
```

Comment:
Again a matter of personal style only: I'd spend another pair of parentheses to get rid of the "line continuations"...